doc/cephfs/troubleshooting.rst

```diff
@@ -2,17 +2,110 @@
  Troubleshooting
 =================
 
+Slow/stuck operations
+~~~~~~~~~~~~~~~~
+If you are experiencing apparent hung operations, the first task is to identify
+where the problem is occurring: in the client, the MDS, or the network connecting
+them. Start by looking to see if either side has stuck operations, and narrow it
+down from there.
```

Comment:
+1 - any pointer for how to diagnose this would be helpful, even if it's a link to another doc.